pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/KafkaConnectSource.java

```diff
@@ -188,15 +198,48 @@ public void close() {
         @Getter
         Optional<String> destinationTopic;
 
+        private final AvroData avroData;
+
+        private org.apache.pulsar.kafka.shade.avro.Schema keyAvroSchema;
+
+        private org.apache.pulsar.kafka.shade.avro.Schema valueAvroSchema;
+
+        private final KafkaSchema keySchema;
+
+        private final KafkaSchema valueSchema;
+
+        private byte[] keyBytes;
```

Comment:
why do we need to make this variable a class variable? it is only used in the constructor.